### PR TITLE
Minor TS fixes, fixed a few breaking imports

### PR DESCRIPTION
**Pull request by corpulent**

### CODE REVIEW

File: packages/console/src/screens/app-builder/panels/explorer/manager/LayerManagerProvider.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { useMethods } from "../../../../../craft-utils";
 import { LayerEventContextProvider } from "../events";
 import { LayerOptions } from "../interfaces";
-import { DefaultLayer } from "../layers";
+import { DefaultLayer } from "../layers/index";
```
Comment (itssamuelrowe) [inline packages/console/src/screens/app-builder/panels/explorer/manager/LayerManagerProvider.tsx]: This seems a little odd. Shouldn't TS/Webpack pickup `../layers/index.ts` via `../layers`?

File: packages/console/src/screens/app-builder/panels/explorer/Layers.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import type { LayerOptions } from "./interfaces";
 import { LayerContextProvider } from "./layers/LayerContextProvider";
 import { LayerManagerProvider } from "./manager/LayerManagerProvider";
 
-export { useLayer } from "./layers";
+export { useLayer } from "./layers/index";
```
Comment (itssamuelrowe) [inline packages/console/src/screens/app-builder/panels/explorer/Layers.tsx]: This seems a little odd. Shouldn't TS/Webpack pickup `../layers/index.ts` via `../layers`?

File: packages/common/tsconfig.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
         "outDir": "dist",
         "allowJs": true,
         "declaration": true,
-        "declarationMap": true
+        "declarationMap": true,
+        "skipLibCheck": true
```
Comment (itssamuelrowe) [inline packages/common/tsconfig.json:12]: We might wanna disable this eventually.

Comment (itssamuelrowe) [reply packages/common/tsconfig.json:12]: I'm making a note of this.

File: packages/common/source/utils/controller.ts [hunk 4]
```diff
@@ -76,12 +76,12 @@ export const createHelper = <T, E>(
             };
         },
 
-        listByIds: async (context, ids: string[]): Promise<E[]> => {
+        listByIds: async (context: any, ids: string[]): Promise<E[]> => {
```
Comment (itssamuelrowe) [inline packages/common/source/utils/controller.ts:79]: This file is legacy. It is removed in my current PR, I think.

File: packages/api/source/controllers/user.controllers.ts [hunk 5]
```diff
@@ -470,7 +470,7 @@ const loginWithGoogle = async (
             emailVerified,
             birthday: null,
             status: "activated",
-        });
+        }) as any;
```
Comment (itssamuelrowe) [inline packages/api/source/controllers/user.controllers.ts:473]: Why does this need to be cast as `any`?

**Review by itssamuelrowe — DISMISSED** [review]

**Review by corpulent — COMMENTED** [review]
Good to merge.

**Pull request merged**